src/EA.Weee.Database/scripts/Update/Release 8/Sprint2/20250127-1254-alter-Evidence-vwTransferTonnageSummary.sql

```diff
@@ -117,7 +117,7 @@ FROM
 						ntt.Reused
 					) AS t
 					PIVOT
-					(   AVG(ReusedTransferred) FOR CategoryId IN ([1], [2], [3], [4], [5], [6], [7],	[8], [9], [10], [11], [12], [13], [14], [15])
```

Comment:
thanks for fixing this tab; no idea why it was there, but I knew it wasn't your doing so I hadn't objected!